src/OpenSage.Game/Logic/Object/Draw/W3dModelDraw.cs

```diff
@@ -245,7 +287,7 @@ internal override void SetWorldMatrix(in Matrix4x4 worldMatrix)
 
     internal sealed class W3dModelDrawConditionState : DisposableBase
     {
-        private readonly ModelInstance _modelInstance;
+        public readonly ModelInstance _modelInstance;
```

Comment:
If you need to access this from the outside the class, please rename it to `ModelInstance`.

Reply:
Was fixed by someone else adding a public Model